Assets/Scripts/Minigames/CowardMiniGame/GearCompletionCheck.cs

```diff
@@ -12,40 +13,35 @@
 public class GearCompletionCheck : MonoBehaviour
 {
     [Header("CheckList")]
-    [SerializeField] private GameObject[] _realGears;
-    private bool _isGameComplete;
-    private int _greenCount;
-    [SerializeField] private Renderer[] _matCheck;
+    [SerializeField] private GameObject[] _gearSlots;
```

Comment:
nit: Can you go through these files and group the serialized and private vars together?